pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -56,6 +64,71 @@ func (ctrl *Controller) handleNodeConfigErr(err error, key string) {
 	ctrl.nodeConfigQueue.AddAfter(key, 1*time.Minute)
 }
 
+// createAutoSizingIgnConfig creates the Ignition config for auto-sizing disabled
+func createAutoSizingIgnConfig() ([]byte, error) {
+	autoSizingFile := ctrlcommon.NewIgnFileBytes("/etc/node-sizing-enabled.env", []byte(DefaultAutoSizingEnvContent))
+	autoSizingIgnConfig := ctrlcommon.NewIgnConfig()
+	autoSizingIgnConfig.Storage.Files = append(autoSizingIgnConfig.Storage.Files, autoSizingFile)
+	rawAutoSizingIgn, err := json.Marshal(autoSizingIgnConfig)
+	if err != nil {
+		return nil, err
+	}
+	return rawAutoSizingIgn, nil
+}
+
+// newAutoSizingMachineConfig creates an empty auto-sizing MachineConfig for a given pool
+func newAutoSizingMachineConfig(pool *mcfgv1.MachineConfigPool) (*mcfgv1.MachineConfig, error) {
+	autoSizingDisabledName := fmt.Sprintf("01-%s-auto-sizing-disabled", pool.Name)
```

Comment:
nit: Just to improve the code readability

```suggestion
	autoSizingDisabledMCName := fmt.Sprintf("01-%s-auto-sizing-disabled", pool.Name)
```